src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureSelector.h

```diff
@@ -94,6 +100,28 @@ namespace OpenMS
     bool   select_highest_count_;
     String variable_type_;
     double optimal_threshold_;
+    Int _addVariable(LPWrapper& problem, String& name, bool bounded, double obj);
+    void _addConstraint(LPWrapper& problem, size_t size, Int *indices_array, double *values_array, String name, double lb, double ub, LPWrapper::Type param);
```

Comment:
If it is possible to use references (instead of pointers), it would be beneficial to take advantage of them. They provide safer code.
If not possible, what is the reason?